src/a-d/Adx/Adx.StreamHub.cs

```diff
@@ -1,18 +1,12 @@
 namespace Skender.Stock.Indicators;
 
 /// <summary>
-/// Represents a stream hub for calculating the Average Directional Index (ADX).
+/// Streaming hub for calculating Average Directional Index (ADX).
```

Comment:
I think I actually prefer this variant form for phrasing these top level classes, for all StreamHub indicators.  Let's keep it simple.

```suggestion
/// Streaming hub for Average Directional Index (ADX)
```